.gitignore

```diff
@@ -1,9 +1,4 @@
 local.properties
-/buildSrc/build/
-/popkorn/build/
-/popkorn-compiler/build/
-/popkorn-example/build/
-/.idea/
-/.gradle/
-/buildSrc/.gradle/
-/build/
+.idea/
+.gradle/
+build/
```

Comment:
With this, we can avoid adding `build` or `.gradle` subfolders every time we create a new module.